pages/api/performance/get.js

```diff
@@ -0,0 +1,28 @@
+import prisma from '@prisma/index'; // Prisma client
+import { getSession } from 'next-auth/client'; // Session handling
+
+// get performances by specifying the corresponding eventID
+// POST with { eventID: id }
+export default async (req, res) => {
+  // Collect session from request
+  const session = await getSession({ req });
+
+  const { id: id } = req.body;
```

Comment:
No need to rename this field when destructuring.
```suggestion
  const { id } = req.body;
```